### PR TITLE
chore: 124 type and stratify in playground

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
-Cleaning up a bunch of now redundant code.
-Much better call to model-service for creating models (1 instead of n)
-Ensuring stratify functionality works with these updates

https://user-images.githubusercontent.com/17088680/207646939-1f612d20-b1b6-499f-b5c3-09785fb75584.mov

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves #(issue)
#124 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Manually + very base level yarn test

# Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/client/hmi-client/src/views/TA2Playground.vue [hunk 1]
```diff
@@ -695,74 +607,58 @@ export default defineComponent({
 				const modelData = await resp.data;
 				modelId = modelData.id;
 				console.log(`Model ID: ${modelId}`); // currently required for testing
-			}
 
-			// Reset current nodes and edges
-			g.nodes = [];
-			g.edges = [];
+				const juliaNodes: { name: string; type: string }[] = [];
+				const juliaEdges: { source: string; target: string }[] = [];
```
Comment (YohannParis) [inline packages/client/hmi-client/src/views/TA2Playground.vue]:
```suggestion
				const modelNodes: { name: string; type: string }[] = [];
				const modelEdges: { source: string; target: string }[] = [];
```

File: packages/client/hmi-client/src/views/TA2Playground.vue [hunk 2]
```diff
@@ -695,74 +607,58 @@ export default defineComponent({
...
-				const transitionID = `t-${iEdges.it}`;
-				this.addEdgeID(sourceID, transitionID, createFlag);
-			}
-			for (let i = 0; i < model.O.length; i++) {
-				const oEdges = model.O[i];
-				const sourceID = `s-${oEdges.os}`;
-				const transitionID = `t-${oEdges.ot}`;
-				this.addEdgeID(transitionID, sourceID, createFlag);
+				// add Nodes
+				for (let i = 0; i < model.S.length; i++) {
```
Comment (YohannParis) [inline packages/client/hmi-client/src/views/TA2Playground.vue:615]: All those for loop could be `Array.map()`

File: packages/client/hmi-client/src/views/TA2Playground.vue (hunk 1 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/views/TA2Playground.vue]: Good point

File: packages/client/hmi-client/src/views/TA2Playground.vue (hunk 2 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/views/TA2Playground.vue]: Done - hope more readable now!

**Review by YohannParis — APPROVED** [review]
LGTM

**Pull request merged**